codegen/templates/tchannel_client.tmpl

```diff
@@ -161,6 +159,9 @@ type {{$clientName}} struct {
 			return respHeaders, err
 		{{else -}}
 			resp, err = {{.GenCodePkgName}}.{{title $svc.Name}}_{{title .Name}}_Helper.UnwrapResponse(&result)
+			if err != nil {
```

Comment:
not all errors are `.Error()` ; default to `.Warn()` here instead.

Reply:
Done.